godtools/App/Features/UserLessonProgress/Domain/Interface/GetResumeLessonProgressModalInterfaceStringsRepositoryInterface.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  GetResumeLessonProgressModalInterfaceStringsRepositoryInterface.swift
```

Comment:
Hey @levieggertcru, how do you feel about the naming convention making some names super long?  Was debating to take out the word "progress", but still gonna be pretty long either way 🤔

Reply:
Hey @rachaelblue I know some names get super long.  I know we suffix a lot of key names such as UseCase, DomainModel, RepositoryInterface, etc. 

And some names get descriptive based on feature so there isn't collisions with other features. (If we could namespace that would help here). 

I think we could start omitting Repository from the interface naming.  

Maybe ```GetResumeLessonInterfaceStringsInterface``` or even ```GetResumeLessonStringsInterface```.  A tiny bit shorter.